pbin.sh

```diff
@@ -23,6 +23,17 @@
 set -e
 PROGRAM=${0##*/}
 
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
+  AWKPROG=awk
+  GOPTPROG=getopt
+elif [[ "$OSTYPE" == "darwin"* ]]; then
+  AWKPROG=gawk
```

Comment:
i think can change this to `gawk` unconditionally regardless of OS, do not need variables then